Modules/_cursesmodule.c

```diff
@@ -4732,9 +4732,28 @@ PyInit__curses(void)
         return NULL;
     ModDict = d; /* For PyCurses_InitScr to use later */
 
+    void **PyCurses_API = PyMem_Calloc(PyCurses_API_pointers, sizeof(void *));
+    if (PyCurses_API == NULL) {
+        PyErr_NoMemory();
+        return NULL;
+    }
+    /* Initialize the C API pointer array */
+    PyCurses_API[0] = (void *)&PyCursesWindow_Type; // borrowed ref
```

Comment:
Would it be possible to hold a strong reference and call Py_DECREF() in curses_destructor?

Reply:
Of course, updated.